advanced/Scripts/query.sh

```diff
@@ -128,22 +134,16 @@ Main() {
     # Test if the authentication endpoint is available
     TestAPIAvailability
```

Comment:
This can be removed as `TestAPIAvailability` is now called from within `LoginAPI`